### DISCUSSION TITLE
What is the point of using CC2530/CC2531 routers ?

### DISCUSSION THREAD

**Discussion by francisp2** [Koenkk/zigbee2mqtt#8518] [Devices]
I have never seen an end-device connecting through them, only other routers

![image](https://user-images.githubusercontent.com/52328611/130824863-20ae508f-809d-4d83-aca5-cb18932d5421.png)

**Comment by Tediore** [comment]
You have a very high ratio of routers to end devices, and it looks like the routes that the network has currently decided are optimal simply just don't involve those CC253x having children. There's always a chance that some end devices could pick one of those CC253x as its parent in the future though. Plus, despite not having any children, those CC253x could still be routing traffic back to the coordinator from one of the non-CC253x routers, but it's hard to tell from the very dense map.

**Comment by AbeltjeNL** [comment]
My advice, don't use them. Recently i removed 5 of these and my network is much more stable and responsive. 
End devices kept switching between CC2531 and other routers which resulted in unresponsiveness.
Keep in mind, the CC2531 router firmware is based on the Zigbee 1.2 protocol. Many other devices these days support 3.0. Should play nice together but is not always the case.
